src/server/cli/test.js

```diff
@@ -0,0 +1,159 @@
+import { ConfigLoaderError, readConfig } from '@web/config-loader';
+import commandLineArgs from 'command-line-args';
+import commandLineUsage from 'command-line-usage';
+import process from 'node:process';
+import { startTestRunner } from '@web/test-runner';
+import { WTRConfig } from '../wtr-config.js';
+
+async function getTestRunnerOptions(argv = []) {
+
+	const DISALLOWED_OPTIONS = ['--browsers', '--playwright', '--puppeteer', '--groups'];
+
+	const optionDefinitions = [
+		// @web/test-runner options
+		{
+			name: 'files',
+			type: String,
+			multiple: true,
+			description: 'Test files to run. Path or glob.\n[Default: ./test/**/*.<group>.js]',
+			order: 8
+		},
+		{
+			name: 'group',
+			type: String,
+			required: true,
+			defaultOption: true,
+			description: 'Name of the group to run tests for\n[Default: test]',
+			order: 1
+		},
+		{
+			name: 'manual',
+			type: Boolean,
+			description: 'Starts test runner in manual testing mode. Ignores browser options and prints manual testing URL.\n{underline Not compatible with automated browser interactions}\nConsider using --watch to debug in the browser instead',
+			order: 11
+		},
+		{
+			name: 'watch',
+			type: Boolean,
+			description: 'Reload tests on file changes. Allows debugging in all browsers.',
+			order: 9
+		},
+
+		// d2l-test options
+		{
+			name: 'chrome',
```

Comment:
One thing I was thinking about - does this affect the browser names the runner has?  Will the goldens be stored in `chrome` and `safari` folders, or still `chromium` and `webkit`?

Reply:
Nothing else changes. I did start second-guessing this because literally everything else will not match these. Directories, test output, report tabs, (playwright/wtr options).

Reply:
Yeah, it's convenient but makes me like it a bit less if everything else will still say the old name

Reply:
I'd be happy to switch what the report shows if that's what we're worried about. The output in the console though will still say "chromium" and "webkit". Surprised Firefox doesn't resort to "gecko".

Reply:
I think the weirdest part for me would be the folders not matching, which I guess we could change by having a map in the plugin.